packages/plugins/typed-variable-modal/test/typed_variable_modal_test.mocha.js

```diff
@@ -0,0 +1,273 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Unit tests for TypedVariableModal.
+ * @author aschmiedt@google.com (Abby Schmiedt)
+ */
+
+const assert = require('assert');
+const Blockly = require('blockly');
+const sinon = require('sinon');
+
+const TypedVariableModal = require('../dist/index.js').TypedVariableModal;
```

Comment:
This is not a this PR question but in general, why are we including the dist version (rather than including src/index.js)
Avoiding the need for a build before running these tests.

Reply:
So just tried to change this to src/index.js and I think the answer is that we need the umd build in order to run in node.

Reply:
Ah makes sense.